x-pack/legacy/plugins/siem/public/components/formatted_date/index.tsx

```diff
@@ -62,3 +63,36 @@ export const FormattedDate = pure<{
 );
 
 FormattedDate.displayName = 'FormattedDate';
+
+/**
+ * Renders the specified date value according to under/over one hour
+ * Under an hour = relative format
+ * Over an hour = in a format determined by the user's preferences,
+ * with a tooltip that renders:
+ * - the name of the field
+ * - a humanized relative date (e.g. 16 minutes ago)
+ * - a long representation of the date that includes the day of the week (e.g. Thursday, March 21, 2019 6:47pm)
+ * - the raw date value (e.g. 2019-03-22T00:47:46Z)
+ */
+
+export const FormattedRelativePreferenceDate = ({ value }: { value?: string | number | null }) => {
```

Comment:
To`FormattedRelativePreferenceDate` or to `RelativeFormattedPreferenceDate`? That is the question!

Seems there's [some dispute](https://en.wikipedia.org/wiki/Adjective#Order) around if the Age modifier should be before or after Shape. Good to know us engineers aren't the only ones that care deeply about specificity... 😅


_(I just learned this recently, so just a shameless plug for that wiki article -- no change necessary!)_

Reply:
Quite